plasmapy/formulary/distribution.py

```diff
@@ -6,6 +6,7 @@
 import astropy as astropy
 from astropy import units as u
 from plasmapy.formulary import parameters
```

Comment:
As a practice we should start importing only the functions we need for the module.  This is already a practice we want to start putting forward in our documentation and it will mitigate the loading of unnecessary objects for users.

```suggestion
from plasmapy.formulary.parameters import (
    kappa_thermal_speed,
    thermal_speed,
}
```

Reply:
Does it actually load just the two functions in the latter case? Might be tough to benchmark this...

Reply:
From my recent research, yes.  Look at the examples section for [python's documentation on the `import` statement.](https://docs.python.org/3/reference/simple_stmts.html#the-import-statement).  I've checked this behavior by examining the module level namespace, and only `kappa_thermal_speed` and `thermal_speed` would be present.  Now, does Python import the whole module and only binds the function to the namespace?  That might happen, but would require digging into the `import` code to pindown (I'm not volunteering for that!!!) and the Python documentation suggests that is not the case.

Reply:
I think there is no performance difference between the two ways: https://softwareengineering.stackexchange.com/questions/187403/import-module-vs-from-module-import-function

Based on very little research, python loads the entire module, but only binds the listed methods to the namespace.

Reply:
Either way, I'm willing to completely neglect the time spent importing stuff until benchmarks show it's becoming a problem :laughing: and I wouldn't bother enforcing a particular import style, either.

Reply:
Resolving this :).